.github/workflows/linux.yml

```diff
@@ -163,13 +171,26 @@ jobs:
           sudo apt update
           sudo apt install -y libxkbcommon-dev libmysqlclient21
 
+      # TODO: Figure out why this error only occurs on the Linux container when building with -DGIT_REVISION=ON
+      # The error: fatal: detected dubious ownership in repository
+      - name: Prevent git's dubious ownership message
+        if: github.ref_type != 'tag'
+        run: |
+          git config --global --add safe.directory "${PWD}"
+
+      # TODO: Migrate to the Ninja generator once we drop support for Ubuntu 18.04.
```

Comment:
Does the Ninja generator not work on 18.04?

Reply:
It does, but because Ubuntu 18.04 has an ancient version of CMake that doesn't support either `--install` and `--prefix`, I couldn't figure out a way to set the prefix on the `Install` step, which on macOS (for example), is invoked like this: `cmake --install --prefix foo`.

To be fair, I'm not even sure if that's the correct command, lol...

I really want to use the Ninja generator across all OSes, but Ubuntu 18.04 is kinda making that a PITA, so let me know if you know how to properly do that. Thanks!

Reply:
No idea, let's just wait until we stop supporting 18.04 lol

Reply:
Right. lol. Just a few more months until official support is dropped... 🤞